### ISSUE TITLE
Issue deploying Tornjak Helm chart with Minikube

### ISSUE THREAD

**Issue by maia-iyer** [IBM/trusted-service-identity#162]
For some reason the `ghcr.io/spiffe/tornjak-fe:latest` image fails to pull. On `helm install... ` i get `ErrImagePull` repeatedly. 

After `minikube ssh` and then manually pulling the image `docker pull ghcr.io/spiffe/tornjak-fe:latest` I do not get this issue anymore. The pull takes a while, so my guess is it times out. 

After this happens, looking into the logs for the frontend container, I get to a line that says `Starting development server...` but then, the container stops mysteriously.  When I use a different optimized image, this does not happen, although that image does require a restart.  This is likely more a problem with the image itself than the deployment. 

tl;dr: there is an ErrImagePull issue with Minikube specifically
